app/javascript/packs/App.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Body from './Body'
+import MyHeader from './MyHeader'
+
+class App extends React.Component {
+
+    constructor() {
+        super();
+        this.state = {
+            cuisines: []
+
+        };
+    }
+
+    componentWillMount() {
+        fetch('http://localhost:3000/cuisines.json')
+            .then(response => response.json())
+            .then((cuisines) => this.setState({cuisines: cuisines}));
+    }
+
+    render() {
+        console.log(this.props.cuisines)
+        return (
+            <div className="app">
+                { this.state.cuisines &&
+                    <MyHeader cuisines = {this.state.cuisines}/>
```

Comment:
No spaces before and after the `=`

Check-out [our style-guide](https://github.com/WeConnect/style-guide/) and consider installing a linter (highly recommended).